source/transfer_list.rst

```diff
@@ -245,10 +245,11 @@ Anyone is free to use tags from that range for any custom TE layout without
 adding their definitions to this specification first. The use of this range is
```

Comment:
In the first paragraph of this section: "Submitters should try to group tag IDs..."
"... may use adjacent tag numbers ..."
In the 2nd paragraph: "Anyone may use tags ..."

Reply:
Done